libs/partners/ai21/langchain_ai21/embeddings.py

```diff
@@ -0,0 +1,40 @@
+from typing import List, Any
+
+from ai21.models import EmbedType
+
+from langchain_ai21.ai21_base import AI21Base
+from langchain_core.embeddings import Embeddings
+
+
+class AI21Embeddings(Embeddings, AI21Base):
+    """AI21 Embeddings embedding model.
+    To use, you should have the 'AI21_API_KEY' environment variable set
+    or pass as a named parameter to the constructor.
+
+    Example:
+        .. code-block:: python
+
+            from langchain_ai21 import AI21Embeddings
+
+            embeddings = AI21Embeddings()
+            query_result = embeddings.embed_query("Hello embeddings world!")
+    """
+    def embed_documents(self, texts: List[str], **kwargs: Any) -> List[List[float]]:
+        """Embed search docs."""
+        response = self.client.embed.create(
+            texts=texts,
+            type=EmbedType.QUERY,
```

Comment:
Shouldn't this be Document?

Reply:
What do you mean? 
Embedding can only get query/segment. If you mean the doc string then im not sure I get it

Reply:
I mean to ask if you are using here the EmbedType that should be used for embedding a document (for indexing)